frontend/demo/src/assets/scss/third-party/lato.scss

```diff
@@ -0,0 +1,73 @@
+/* lato-regular - latin */
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 400;
+  src: url("/fonts/lato-v15-latin-regular.eot");
+  /* IE9 Compat Modes */
+  src: local("Lato Regular"), local("Lato-Regular"),
+    url("/fonts/lato-v15-latin-regular.eot?#iefix") format("embedded-opentype"),
+    /* IE6-IE8 */ url("/fonts/lato-v15-latin-regular.woff2") format("woff2"),
+    /* Super Modern Browsers */ url("/fonts/lato-v15-latin-regular.woff")
+      format("woff"),
+    /* Modern Browsers */ url("/fonts/lato-v15-latin-regular.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */ url("/fonts/lato-v15-latin-regular.svg#Lato")
+      format("svg");
+  /* Legacy iOS */
+}
+
+/* lato-semibold - latin */
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 500;
+  src: url("/fonts/lato-semibold.eot");
+  /* IE9 Compat Modes */
+  src: local("Lato Semibold"), local("Lato-Semibold"),
+    url("/fonts/lato-semibold.eot?#iefix") format("embedded-opentype"),
```

Comment:
I don't think we include these files anymore. If this is also on platform we should also remove it there, because we don't have these files anymore I think.